### DISCUSSION TITLE
Upload is so weak

### DISCUSSION THREAD

**Discussion by asoMansoury** [vpnhood/VpnHood#269] [General]
Can you give any version to make it faster when users are going to upload data especially on social media? 
actually right now the uploading side is really weak, at least for me it is really low.

**Comment by bobvhood** [comment]
Hi, We don't offer any private servers. I am afraid to say that you need to setup one on your own.

**Comment by asoMansoury** [comment]
hi I have my own private server , but actually it seems uploading side is
not work so good.
thanks for your help
